tests/src/test/scala/system/basic/WskBasicTests.scala

```diff
@@ -88,7 +88,10 @@ class WskBasicTests
 
     it should "reject unauthenticated access" in {
         implicit val wskprops = WskProps("xxx") // shadow properties
-        val errormsg = "The supplied authentication is invalid"
+        //First it uses auth key verification, if failed, uses client certificate verification,
+        //for client certificate verification, if header doesn't include x-ssl-subject,it will
+        //throw CredentialsMissing Exception
+        val errormsg = "The resource requires authentication, which was not supplied with the request"
```

Comment:
Yes, you are right, it is in `Spray-routing_2.11-1.3.4-sources.jar`'s RejectionHandler.scala
```
      val rejectionMessage = cause match {
        case CredentialsMissing  ⇒ "The resource requires authentication, which was not supplied with the request"
        case CredentialsRejected ⇒ "The supplied authentication is invalid"
      }
```
I wil add it to singleton object